concord_cli/deploy.py

```diff
@@ -107,14 +110,9 @@ def parseFile(filename, parser):
     return conf
 
 def generate_options():
```

Comment:
We should do some pre parsing on the arguments and see if the first argument **is** the ocnfig file ? 

` concord deploy  operator.json`